example/starwars/starwars.go

```diff
@@ -332,7 +328,13 @@ func (r *Resolver) Character(args struct{ ID graphql.ID }) *characterResolver {
 }
 
 func (r *Resolver) Human(args struct{ ID graphql.ID }) *humanResolver {
+	startTime := time.Now()
+	for _, h := range humans {
```

Comment:
Just a note: I would leave the initialization logic to the `init` function, where it executes only when the application starts, rather than on every request.